acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/ContentSteps.java

```diff
@@ -223,4 +230,41 @@ public void theUploadIsRejectedWithAnErrorMessage() throws Throwable {
                 contentPage.getErrorMessage(), startsWith(expectedErrorMessage));
         });
     }
+
+    @Given("^I have a number o publications belonging to the same time series$")
+    public void iHaveANumberOPublicationsBelongingToTheSameTimeSeries() throws Throwable {
+
+        final TimeSeries timeSeries = TestDataLoader.loadValidTimeSeries().build();
+        testDataRepo.setCurrentTimeSeries(timeSeries);
+    }
+
+    @When("^I navigate to the time series$")
+    public void iNavigateToTheTimeSeries() throws Throwable {
+        consumablePublicationSeriesPage.open(testDataRepo.getCurrentTimeSeries());
+    }
+
+    @Then("^I can see a list of released publications from the time series$")
+    public void iCanSeeAListOfPublicationsFromTheTimeSeries() throws Throwable {
+
+        final TimeSeries expectedTimeSeries = testDataRepo.getCurrentTimeSeries();
+
+        assertThat("Correct publication time series title is displayed.",
+            consumablePublicationSeriesPage.getSeriesTitle(), is(expectedTimeSeries.getTitle()));
+
+
+        final List<Publication> expectedPublications = expectedTimeSeries.getPublications().stream()
+            .filter(publication -> PUBLISHED.equals(publication.getStatus()))
+            .collect(toList());
+
+        // Sort expected publications with latest nominal dates at the top
+        expectedPublications.sort((left, right) -> -1 * left.getNominalDate().compareTo(right.getNominalDate()));
```

Comment:
Could/should we push this sorting down into the TimeSeries? I know it's not an intrinsic property of the time series, as such, but it feels a bit laborious having it here.

Reply:
Indeed, having this method in TimeSeries does read better and I have now moved it there.